include/roboteam_ai/utilities/StrategyManager.h

```diff
@@ -28,40 +28,47 @@ class StrategyManager {
     const std::vector<RefGameState> gameStates = {
 
         // failsafe: for an undefined refstate everything should halt
-        RefGameState(RefCommand::UNDEFINED, "halt_strategy", "keeper_halt_tactic", "halt"),
+        RefGameState(RefCommand::UNDEFINED, "halt", "halt"),
 
-        RefGameState(RefCommand::NORMAL_START, "normal_play_strategy", "keeper_default_tactic", "default"),
-        RefGameState(RefCommand::FORCED_START, "normal_play_strategy", "keeper_default_tactic", "default"),
-        RefGameState(RefCommand::HALT, "halt_strategy", "keeper_halt_tactic", "halt"), RefGameState(RefCommand::STOP, "stop_strategy", "keeper_avoid_tactic", "stop"),
-        RefGameState(RefCommand::TIMEOUT_US, "time_out_strategy", "keeper_formation_tactic", "stop"),
-        RefGameState(RefCommand::TIMEOUT_THEM, "halt_strategy", "keeper_halt_tactic", "halt"),
-        RefGameState(RefCommand::GOAL_US, "kickoff_them_formation_strategy", "keeper_formation_tactic", "default"),
-        RefGameState(RefCommand::GOAL_THEM, "kickoff_us_formation_strategy", "keeper_formation_tactic", "default"),
-        RefGameState(RefCommand::BALL_PLACEMENT_US, "ball_placement_us_strategy", "keeper_avoid_tactic", "ballplacement_us"),
-        RefGameState(RefCommand::BALL_PLACEMENT_THEM, "ball_placement_them_strategy", "keeper_avoid_tactic", "ballplacement_them"),
-        RefGameState(RefCommand::DIRECT_FREE_US, "free_kick_shoot_strategy", "keeper_default_tactic", "default"),
-        RefGameState(RefCommand::DIRECT_FREE_THEM, "free_kick_them_strategy", "keeper_default_tactic", "default"),
-        RefGameState(RefCommand::INDIRECT_FREE_US, "free_kick_shoot_strategy", "keeper_default_tactic", "default"),
-        RefGameState(RefCommand::INDIRECT_FREE_THEM, "free_kick_them_strategy", "keeper_default_tactic", "default"),
+        RefGameState(RefCommand::NORMAL_START, "normal_play", "default"),
+        RefGameState(RefCommand::FORCED_START, "normal_play", "default"),
+        RefGameState(RefCommand::HALT, "halt", "halt"),
+        RefGameState(RefCommand::STOP, "stop", "stop"),
+        RefGameState(RefCommand::TIMEOUT_US, "time_out", "stop"),
+        RefGameState(RefCommand::TIMEOUT_THEM, "halt", "halt"),
```

Comment:
Why are these not the same?

Reply:
I think because when we have a time out, we do time_out, whatever that is. And when the other team has a time out, we dont have to do something special and we just halt